lib/studies/reply/mail_card_preview.dart

```diff
@@ -26,72 +38,95 @@ class MailPreviewCard extends StatelessWidget {
+                                Text(email.subject, style: textTheme.headline6),
+                                const SizedBox(height: 16),
+                              ],
                             ),
-                            color: ReplyColors.blue600,
                           ),
-                          SizedBox(width: 20),
-                          ImageIcon(
-                            AssetImage(
-                              '$_iconAssetLocation/twotone_delete.png',
-                              package: _assetsPackage,
-                            ),
-                            color: ReplyColors.blue600,
+                        ),
+                        Transform.translate(
+                          offset: const Offset(-20, -20),
+                          child: Row(
+                            children: [
+                              if (isDesktop)
+                                Row(
+                                  children: const [
+                                    ImageIcon(
+                                      AssetImage(
+                                        '$_iconAssetLocation/twotone_star.png',
+                                        package: _assetsPackage,
+                                      ),
+                                      color: ReplyColors.blue600,
+                                    ),
+                                    SizedBox(width: 20),
+                                    ImageIcon(
+                                      AssetImage(
+                                        '$_iconAssetLocation/twotone_delete.png',
+                                        package: _assetsPackage,
+                                      ),
+                                      color: ReplyColors.blue600,
+                                    ),
+                                    SizedBox(width: 20),
+                                    Icon(Icons.more_vert,
+                                        color: ReplyColors.blue700),
+                                  ],
+                                ),
+                              const SizedBox(width: 20),
+                              Transform.translate(
+                                offset: Offset(0, isDesktop ? -16 : 8),
+                                child: ClipOval(
+                                  child: Image.asset(
+                                    email.avatar,
+                                    package: _assetsPackage,
+                                    height: 36,
+                                    width: 36,
+                                  ),
+                                ),
+                              ),
+                            ],
                           ),
-                          SizedBox(width: 20),
-                          Icon(Icons.more_vert, color: ReplyColors.blue700),
-                        ],
+                        ),
+                      ],
```

Comment:
Would benefit in readability for this to be its own StatelessWidget

Reply:
Will do